### ISSUE TITLE
Not an issue, just a question

### ISSUE THREAD

**Issue by kallekutta** [EuFlo/sacd-ripper#10]
Hello,

I'm having trouble understanding the usage of the option -c in sacd_extract. According to the manual it's used to: "convert DST to DSD". I have two questions.

**First question**
How can one know if the sacd iso is in DST? I have tried the option -P (display disc and track information) but it doesn't output that kind of information. Luckily with the help of foobar2000 and the foo_input_sacd plugin it's possible to identify which sacd isos are in DST. But without foobar2000, how can one identify this information and know when to use the -c option?

**Second question**
I did an experiment and extracted the first track of two different albums, one in DSD and the other in DST. I did the extraction to DSF with the -s option. First I extracted the DSD album with and without the -c option. I wanted to see if forcing DST conversion to an album not in DST had any effect. But both files turned out the same, so no difference when using the -c option on DSD albums. Then I tried the same thing with the DST album. This is where it gets weird. Both files also turned out the same. My expectation was that when extracting a DST album without the -c option would result in some kind of error or something. How is this possible? What exactly does the -c option do and when to use it?

**Comment by EuFlo** [comment]
Hello,
Sorry for late answer.
Even this option(and functionality) is created in original master repository, I will try to explain some things here:
Answer of Q1) this app does not have an option to notify the user of the presence (or not) of DST on an iso.
Answer of Q2)  -c option is build only for using in conjunction of dsdiff ( or dsdiff-em) file extraction. This is useless for extracting dsf files. DSF files must be always decompressed (if DST is present on sacd, the decompression is made automatically).

**Comment by kallekutta** [comment]
Thank you very much for this valuable information.

I usually extract to DSF so that means I don't have to worry about the -c option.

However, I got curious to do my experiment again with this new information in mind. So this time I extracted the same two albums to dsdiff (option -p) and I can see a difference now for the DST compressed album. With the -c option it was decompressed and without it no decompression.

The DSD album though was the same with or without the -c option. That means telling the program to decompress a non-compressed album doesn't seem to cause any problem. So if my preferred extraction result is decompressed that means I don't need to know in advance if the album is compressed or not. It should be safe to use the -c option all the time on all types of albums. Is that a correct interpretation?
